orderbook/src/price_estimate.rs

```diff
@@ -138,6 +140,29 @@ impl UniswapPriceEstimator {
         })
     }
 
+    /*
+        Returns a vector of (rational) prices for the given tokens
+        denominated in denominator_token.
+    */
+    pub async fn best_execution_spot_prices(
+        &self,
+        tokens: &Vec<H160>,
+        denominator_token: H160,
+    ) -> Result<Vec<BigRational>> {
+        let res = join_all(tokens.iter().map(|token| {
+            self.best_execution(
```

Comment:
Would you not be able to reuse `self.best_execution_spot_price` here?

Reply:
It seems best_execution_spot_price is returning a float. Not sure what the best option is actually, I was thinking that for comparing solutions we could get away with big rationals and big ints without going into floats. Wdyt?